### PR TITLE
meson: fix incomplete option descriptions

### PR DESCRIPTION
**Pull request by rindeal**

### CODE REVIEW

File: meson_options.txt [hunk 1]
```diff
@@ -144,9 +144,9 @@ option('dbussessionservicedir', type : 'string',
 option('dbussystemservicedir', type : 'string',
        description : 'D-Bus system service directory')
 option('pkgconfigdatadir', type : 'string', value : '',
-       description : 'directory for ')
+       description : 'directory for arch-independent pkg-config files')
```
Comment (yuwata) [inline meson_options.txt:147]: `pkg-config files`?? Maybe, `arch-independent data files` is ok. WDYT?

File: meson_options.txt [hunk 2]
```diff
@@ -144,9 +144,9 @@ option('dbussessionservicedir', type : 'string',
...
 option('pkgconfiglibdir', type : 'string', value : '',
-       description : 'directory for ')
+       description : 'directory for standard pkg-config files')
```
Comment (yuwata) [inline meson_options.txt:149]: Same here. Maybe, `directory for library files`.

File: meson_options.txt (hunk 1 above)
Comment (rindeal) [reply meson_options.txt:147]: pkg-config manpage calls those files _"special metadata files"_, just `data files` is too generic for my taste, `library files` or `library metadata files` would imply it's for information related libraries only, which is not always the case and especially not here. So I just settled with `pkg-config files` for the lack of a good standardized term. `*.pc files` would be my second suggestion. However, I'm not going to bicker about something like this so make your final suggestion please and I'll force-push your re-wording.

Comment (yuwata) [reply meson_options.txt:147]: cc @keszybz.

Comment (rindeal) [reply meson_options.txt:147]: [Meson docs](https://mesonbuild.com/Pkg-config-files.html) refer to those files as "Pkg-config files", i.e. the same I did.

Comment (yuwata) [reply meson_options.txt:147]: Haha, I missed the option is about `pkgconfig`. Sorry, please forget my above comment.

LGTM.

**Pull request merged**